src/frontend/src/pages/ProjectPage/pages/ContractPage/components/RequestWorkflow/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as styles from './styles.less';
+import Workflow from '../../../../../../models/Workflow';
+import WorkflowStep from './WorkflowStep';
+
+type RequestWorkflowProps = {
+    workflow: Workflow;
+    inline?: boolean
+};
+
+const RequestWorkflow: React.FC<RequestWorkflowProps> = ({ workflow, inline }) => {
+    const sortedWorkflowSteps = React.useMemo(() => {
```

Comment:
We should use the prev/next step property to get the correct order of steps. If this changes in the future, we need to update the hardcoded order instead of allowing the backend to control that